app/src/theme/index.tsx

```diff
@@ -524,3 +524,27 @@ export function ThemeProvider({ children }: { children: React.ReactNode }) {
 
   return <StyledThemeProvider theme={themeAggregator(darkMode)}>{children}</StyledThemeProvider>
 }
+
+const TextWrapper = styled.div<{ color: keyof Colors }>`
+  color: ${props => props.theme.text3};
+  font-family: Roboto;
+  letter-spacing: 0.2px;
+`
+
+export const TYPE = {
+  heading1(props: any) {
+    return <TextWrapper fontSize={'22px'} fontWeight={500} letterSpacing={'0.8px'} lineHeight={'26px'} {...props} />
+  },
+  heading2(props: any) {
+    return <TextWrapper fontSize={'18px'} fontWeight={400} lineHeight={'21px'} {...props} />
+  },
+  heading3(props: any) {
+    return <TextWrapper fontSize={'16px'} fontWeight={500} lineHeight={'19px'} {...props} />
+  },
+  bodyMedium(props: any) {
+    return <TextWrapper fontSize={'18px'} fontWeight={500} lineHeight={'16px'} {...props} />
+  },
+  bodyRegular(props: any) {
+    return <TextWrapper fontSize={'18px'} fontWeight={400} lineHeight={'16px'} {...props} />
```

Comment:
You seem to have changed font-size instead of line height! Just fix that and should be gtg! https://github.com/protofire/omen-exchange/pull/2078#discussion_r664815869